bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoMakerHandler.java

```diff
@@ -143,11 +169,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             + "<BinaryState>" + binaryState + "</BinaryState>" + "</u:SetBinaryState>" + "</s:Body>"
                             + "</s:Envelope>";
 
-                    URL descriptorURL = service.getDescriptorURL(this);
-                    String wemoURL = getWemoURL(descriptorURL, "basicevent");
-
-                    if (wemoURL != null) {
-                        wemoCall.executeCall(wemoURL, soapHeader, content);
+                    if (!host.isEmpty()) {
```

Comment:
@hmerk My guess is that this check makes your tests fail (because it block execution of `.executeCall` and is changed from before. I didn't check the code (where is `host` set and could it be that it is empty in the test because it is not initialized?).